src/diffpy/labpdfproc/labpdfprocapp.py

```diff
@@ -29,7 +29,14 @@ def get_args(override_cli_inputs=None):
         "'.' (load everything in the current directory), 'data' (load"
         "everything in the folder ./data), 'data/file_list.txt' (load"
         " the list of files contained in the text-file called "
-        "file_list.txt that can be found in the folder ./data).",
+        "file_list.txt that can be found in the folder ./data). "
+        "\nWildcard character (*) is accepted. Examples include './*.chi'"
+        " (load all files with .chi extension), 'data/*.chi' (load all "
+        "files in 'data' file with .chi extension), 'file*.chi' (load all "
```

Comment:
This is probably a bit too long now.  I htink most people know how "*" wildcards work, so i would just suggest to use the `data/*.chi` example and one example for a directory, maybe `inputs*` which in principle would open all files in any directory that begins with "inputs" but also any file that begins with "inputs" in the current directory.

We may want to think about not letting the users specify directories with "*".  It could cause problems and also not be that useful.  For example, if they just specify "*" it will glob all subdirectories of their current positoin and operate on all files in all the directories.  if they do this from a home directory or something it could give unwanted behavior. 

On balance, I think this might be better, so the wildcard only applies to files in the specified directory.  Then if they want to do this over multiple directories they need to specify this as a list, and if it is a long list it can be done in an `input-files.txt` file.

Reply:
I agree. I simplified and combined the expand functions.